behave/__main__.py

```diff
@@ -95,21 +102,30 @@ def run_behave(config, runner_class=None):
         return 0
 
     if len(config.outputs) > len(config.format):
-        print("CONFIG-ERROR: More outfiles (%d) than formatters (%d)." % \
+        print("CONFIG-ERROR: More outfiles (%d) than formatters (%d)." %
               (len(config.outputs), len(config.format)))
         return 1
 
     if config.runner == "help":
         print_runners(config.runner_aliases)
         return 0
 
+    # -- HANDLE MULTIPROCESSING
+    # Use default parallel runner if custom not provided
+    if config.jobs > 1 and not runner_class:
+        if config.parallel_element == "feature":
+            runner_class = FeatureParallelRunner
+        else:
+            runner_class = ScenarioParallelRunner
+        print("INFO: Number of jobs > 1 - using Parallel Runner: %s" % 
+            runner_class.__name__)
+
     # -- MAIN PART:
     runner = None
     failed = True
     try:
         reset_runtime()
         runner = RunnerPlugin(runner_class).make_runner(config)
-        # print("USING RUNNER: {0}".format(make_scoped_class_name(runner)))
```

Comment:
I'd suggest avoiding making any change that isn't explicitly required to try keep the diff between the fork and the original as small as possible, applies to some of the other changes done